locales/en-US/client.properties

```diff
@@ -27,6 +27,7 @@ renameProjectSavingIndicator=Saving…
 publishPublishingIndicator=Publishing…
 publishUnpublishingIndicator=Unpublishing…
 publishDeleteWarning=Are you sure? The current link will stop working and if you publish again, the link will be different.
+publishHeaderOnline=Your Project is Online
```

Comment:
also move the `publishHeader` string from `server.properties` to `server-client-shared.properties` since we are going to use that string in both the server and the client.